docs/concepts/jcloud/yaml-spec.md

```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
 In JCloud, you can pass highly customizable, finely-grained resource requests for each Executor using the `jcloud.resources` argument in your Flow YAML.
 
 
-### CPU
+### Instance
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
+JCloud uses the concept of "Instance" to respresent the hardware specifications. 
+In above example, C2 Instance Type stands for 4 cores and 8G RAM based on the CPU Tiers Instance defination table below.
 
-JCloud offers the general Intel Xeon processor (Skylake 8175M or Cascade Lake 8259CL) by default. 
-
-```{hint}
-Maximum of 16 cores is allowed per Executor.
-```
+Note that if you are still using the legacy resource specification interface, as such:
 
 ```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        cpu: 0.5
+jcloud:
+  resources:
+    cpu: 8
+    memory: 8G
 ```
 
+We would translate the raw numbers from input to Instance tier that fits most. 
+There are circumstances the Instance tier does not exactly fufill the CPU cores and Memory you need, like in above exmaple. 
+But we would "ceil" the requests to the lowest tier that would satisfy all the specifications. 
+In this case, `C6` would be consisdered, as `C5`'s `Cores` is lower than what's being requested (4 vs 8). 
+
+There are also two types of Instance Tiers, one for CPU instances, one for GPU.
+
+#### CPU Tiers
+
+| Instance | Cores | Memory | Credits Per Hour |
```

Comment:
We introduce `Credits Per Hour` in this table, but do not explain where this is coming from or what they mean. I suggest we introduce the credits concept and/or link to the pricing page.

https://cloud.jina.ai/pricing

Reply:
Yea that's definitely lacking, I will add this piece of info in!

Reply:
I'll check this again after you add that info @zac-li

Reply:
added some